rosbot_xl_gazebo/launch/simulation.launch.py

```diff
@@ -89,24 +89,14 @@ def generate_launch_description():
         executable="parameter_bridge",
         name="ign_bridge",
         arguments=[
-            "/clock" + "@rosgraph_msgs/msg/Clock" + "[ignition.msgs.Clock",
-            "/scan" + "@sensor_msgs/msg/LaserScan" + "[ignition.msgs.LaserScan",
-            "/velodyne_points/points"
-            + "@sensor_msgs/msg/PointCloud2"
-            + "[ignition.msgs.PointCloudPacked",
-            "/camera/color/camera_info"
-            + "@sensor_msgs/msg/CameraInfo"
-            + "[ignition.msgs.CameraInfo",
-            "/camera/color/image_raw"
-            + "@sensor_msgs/msg/Image"
-            + "[ignition.msgs.Image",
-            "/camera/camera_info"
-            + "@sensor_msgs/msg/CameraInfo"
-            + "[ignition.msgs.CameraInfo",
-            "/camera/depth" + "@sensor_msgs/msg/Image" + "[ignition.msgs.Image",
-            "/camera/depth/points"
-            + "@sensor_msgs/msg/PointCloud2"
-            + "[ignition.msgs.PointCloudPacked",
+            "/clock@rosgraph_msgs/msg/Clock[ignition.msgs.Clock",
```

Comment:
Look here for topics remappings:
https://github.com/husarion/ros_components_description/pull/31